INSTALL.md

```diff
@@ -221,7 +221,8 @@ The first script needs to be called in Powershell, and it needs to call `compile
 
     C:\Users\travis\build\AMICI\scripts\compileBLAS.cmd
```

Comment:
@thomassligon : Can you please update the documentation there, taking into account that that script now requires the OpenBLAS version (without "v") as argument. Maybe mentioning that 0.3.12 is known to work.